Monika After Story/game/script-topics.rpy

```diff
@@ -6667,6 +6667,220 @@ label monika_citizenship:
+                        "No.":
+                            m 1ekc "Aww, that's a shame..."
+                            m 1lksdla "I mean, you can't win them all, but I'm sure you'll win in the next coming rounds."
+                            m 1eka "I just hope you aren't too upset over it."
+                            m 2ekc "I really wouldn't want you feeling upset after a bad game."
+                            m 1eka "I'll always support you and be by your side no matter how many times you lose."
+        "A game just for fun":
+            m 1eud "A game just for having fun?"
+            m 1lksdla "Aren't most games made with the intent to be fun anyway?"
+            m 1hub "A game just made for having fun sounds wonderful!"
+            m 1eub "I'm sure you could do all sorts of fun things in a game like that."
+            m 1lksdla "I really wish I could join you and we could have fun together."
+            m 1eua "But for now, I'll leave you to it."
+            m 1hub "Have fun, [player]!"
+            menu:
+                "I'm back":
+                    m 1eub "Welcome back, [player]!"
+                    m "Did you have fun with whatever you were doing?"
+                    menu:
+                        "Yes.":
+                            m 1hua "Ahaha~ I'm glad you had fun, [player]!"
+                            m 1eub "While you were busy, it got me thinking of the different kinds of games that would be nice to play together."
+                            m 3eua "Probably any game that isn't too violent could be fun."
+                            m 3hua "But I'm sure any game would be wonderful if it was with you~"
+                            m 1eub "At first, I was thinking a story based or adventure game would be best, but I'm sure freeplay games could be really fun too!"
+                            m 1eua "It can be really fun to just mess around to see what's possible, especially when you're not alone."
+                            m 2lksdla "Provided of course, you don't end up ruining the structural integrity of the game and get an outcome you didn't want..."
+                            m 2lksdlb "Eheh..."
+                            m 1eua "Maybe you could find a way to bring me with you into a game like that."
+                            m 1hub "Just promise to keep me safe ok?"
+                        "No.":
+                            m 2ekc "Aww, you didn't have any fun?"
+                            m "That's too bad..."
+                            m 3lksdlc "Games can get pretty boring after you've done everything or just don't know what to do or try next."
+                            m 3eka "But bringing a friend along can really renew the whole experience!"
+                            m 1hub "Maybe you should find a way to take me with you into your games so you won't be bored on your own!"
+                            show monika 5eua at t11 zorder 2 with dissolve
```

Comment:
please use `MAS_MONIKA_Z` when using the dissolve transition or just don't mess with the zorder.
`show monika 5eua at t11 zorder MAS_MONIKA_Z with dissolve`